core/pages/Checkout.js

```diff
@@ -258,11 +258,17 @@ export default {
       }
       return paymentMethod
     },
+    prepareOrderProducts () {
+      return this.$store.state.cart.cartItems.map(product => {
```

Comment:
What about making it shorter with?:
```js
return this.$store.state.cart.cartItems.map(product => omit(product, config.orders.excludeFields))
```

https://www.geeksforgeeks.org/lodash-_-omit-method/

Reply:
new PR is created https://github.com/vuestorefront/vue-storefront/pull/6219